catalog.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Калатог - Cat Energy</title>
+  </head>
+  <body>
+    <header class="page-header">
+      <div class="page-header__wrapper">
+        <a class="page-header__logo logo" href="index.html">
+          <img class="logo__image" src="images/logo-mobile.png" width="33" height="37" alt="Логотип Cat Energy.">
+          <img class="logo__modile" src="images/logo-text-mobile.svg" width="101" height="18" alt="логотип мобильный.">
+        </a>
+        <nav class="page-header__main-nav main-nav main-nav--close">
+          <button class="main-nav__toggle" type="button">
+            <span class="visually-hidden">Открыть/закрыть меню</span>
+          </button>
+          <ul class="main-nav__list">
+            <li class="main-nav__item">
+              <a class="main-nav-link main-nav__link--active" href="index.html">Главная</a>
```

Comment:
Давай еще чуть-чуть исправим, блок `main-nav-link` не нужен, у всех ссылок стоит класс `main-nav__link`, соответственно активная ссылка тоже должна иметь такой же класс, и для нижнего подчеркивания еще модификатор active
```suggestion
              <a class="main-nav__link main-nav__link--active" href="index.html">Главная</a>
```